src/test/version.test.ts

```diff
@@ -2,172 +2,77 @@
 
 import * as assert from "assert";
 import * as vscode from "vscode";
-import * as version from "../version";
+
+import { Error } from "../error";
+import { Version } from "../version";
 
 suite("Version Test Suite", () => {
   vscode.window.showInformationMessage("Start all version tests.");
 
   test("Parses simple version successfully", () => {
-    const result = version.parse("0.33.0");
-    const want = {
-      major: 0,
-      minor: 33,
-      patch: 0,
-      releaseCandidate: null,
-    };
+    const result = Version.parse("0.33.0");
+    const want = new Version(0, 33, 0);
     assert.deepStrictEqual(result, want);
   });
 
   test("Parses dev version successfully", () => {
-    const result = version.parse("1.34.15-dev");
-    const want = {
-      major: 1,
-      minor: 34,
-      patch: 15,
-      releaseCandidate: null,
-    };
+    const result = Version.parse("1.34.15-dev");
+    const want = new Version(1, 34, 5);
```

Comment:
Should the patch version here be `15`? Ditto for the test case underneath.

Reply:
This test is gone.